azure-pipelines.yml

```diff
@@ -47,71 +47,3 @@ jobs:
   - publish: _build/log
     artifact: Dune Build Log
     condition: always()
-
-- job: macOS
-  pool:
-    vmImage: 'macOS-10.14'
-
-  variables:
-    MACOSX_DEPLOYMENT_TARGET: '10.11'
-
-  steps:
-
-  - checkout: self
-    fetchDepth: 10
-
-  - script: |
-      set -e
```

Comment:
I didn't keep the `set -e` because I was under the impression that they were not needed (at least on GitHub Actions).